docker/Dockerfile

```diff
@@ -31,4 +31,4 @@ RUN make go-build
 # Container settings.
 ENV LISTEN-ADDRESS 8080
 EXPOSE 8080
-ENTRYPOINT ["./fasttrack-service", "server", "--database-uri=postgres://postgres:postgres@fasttrack-postgres:5432/postgres", "--listen-address=:8080"]
+ENTRYPOINT ["./fasttrack-service", "server", "--database-uri=postgres://postgres:postgres@fasttrack-postgres:5432/postgres", "--listen-address=:8080", "--log-level=debug"]
```

Comment:
Do we want to leave this set at debug?

Reply:
it is pretty useful in general to see debug message and everything what is going on inside the service.

Reply:
so my vote - yes =)

Reply:
this dockerfile only pertains to dev, yes?